modules/atomic-widgets/validators/styles-validator.php

```diff
@@ -38,49 +36,81 @@ public static function make( array $schema ): self {
 	 * }
 	 */
 	public function validate( array $styles ): array {
+		$errors_bag = [];
```

Comment:
It still can be a property member in the class, but you have to reset it after done with the validation.

Reply:
I feel like it is easier to track changes that way